### PR TITLE
Replace 'Finish Review' and 'Delete Review' with an 'Open Review' buton

### LINKED ISSUE #2367: Unable to submit a review with one comment
- Extension version: v2021.1.26474
- VSCode Version: 1.53.0-insider
- OS: Windows 10

Steps to Reproduce:

1. Start a review and add a comment
2. There is no way to submit the review without leaving a second comment

https://user-images.githubusercontent.com/686963/105080342-4a968d80-5a56-11eb-8f14-ade531669aa9.mp4

Linking https://github.com/microsoft/vssaas-planning/issues/2176

### PR DESCRIPTION
**Pull request by RMacfarlane**
Fixes #2367

Fixes https://github.com/microsoft/vscode-pull-request-github/issues/2613

### CODE REVIEW

File: src/github/folderRepositoryManager.ts [hunk 1]
```diff
@@ -141,8 +141,9 @@ export class FolderRepositoryManager implements vscode.Disposable {
 	readonly onDidChangeAssignableUsers: vscode.Event<IAccount[]> = this._onDidChangeAssignableUsers.event;
 
 	constructor(
+		public context: vscode.ExtensionContext,
 		private _repository: Repository,
-		private readonly _telemetry: ITelemetry,
+		public readonly _telemetry: ITelemetry,
```
Comment (alexr00) [inline src/github/folderRepositoryManager.ts:146]: nit: maybe the underscore on `_telemetry` should be romoved now that its public.

File: src/view/pullRequestCommentController.ts [hunk 2]
```diff
@@ -389,19 +390,15 @@ export class PullRequestCommentController implements CommentHandler, CommentReac
...
-			vscode.window.showErrorMessage(`Failed to submit the review: ${e}`);
-		}
-	}
 
-	public async deleteReview(): Promise<void> {
-		await this.pullRequestModel.deleteReview();
-		this.setContextKey(false);
+	public async openReview(): Promise<void> {
+		await PullRequestOverviewPanel.createOrShow(this._folderReposManager.context.extensionUri, this._folderReposManager, this.pullRequestModel);
+		PullRequestOverviewPanel.openReview();
```
Comment (alexr00) [inline src/view/pullRequestCommentController.ts:396]: Why is a call to `openReview` needed? Does `createOrShow` not actually show?

**Review by alexr00 — APPROVED** [review]

File: src/view/pullRequestCommentController.ts (hunk 2 above)
Comment (RMacfarlane) [reply src/view/pullRequestCommentController.ts:396]: Renamed to `scrollToReview` for clarity

File: src/github/folderRepositoryManager.ts (hunk 1 above)
Comment (RMacfarlane) [reply src/github/folderRepositoryManager.ts:146]: thanks, updated

**Pull request merged**